ethcontract/Cargo.toml

```diff
@@ -25,6 +25,7 @@ ws-tokio = ["web3/ws-tokio"]
 ws-tls-tokio = ["web3/ws-tls-tokio"]
 
 [dependencies]
+arrayvec = "0.5"
```

Comment:
I don't see this being used anywhere? Maybe I missed something.

Reply:
It's used in `tokens.rs`.

Reply:
The `token.rs` wasn't expanded when I did Ctrl-F :stuck_out_tongue: